examples/bank/bank.go

```diff
@@ -137,20 +87,23 @@ func (bank *Bank) continuousMoneyTransfer() {
 		exchangeAmount := rand.Int63n(100)
 		// transferMoney transfers exchangeAmount between the two accounts
 		// using transaction txn if its non nil
-		transferMoney := func(txn *client.Txn) error {
-			fromGet := client.Get(from)
-			toGet := client.Get(to)
-			if txn != nil {
-				if err := txn.Run(fromGet, toGet); err != nil {
+		transferMoney := func(tx *client.Tx) error {
```

Comment:
`client/db.go` defines a `runner` interface that is implemented by both `DB` and `Tx`. I think you should export that type (e.g. `s/runner/Runner/g`) and then change `transferMoney` to take a `Runner` instead of a potentially nil `tx`. Then you call this as:

```
  if err := bank.db.Tx(func(tx *client.Tx) error { return transferMoney(tx) }); err != nil {
  ...
  }

// or

  if err := transferMoney(bank.db); err != nil {
    ...
  }
```